src/commands/build.ts

```diff
@@ -45,11 +45,12 @@ export class BuildCommand implements Command {
           'used for service worker generation.'
     },
     {
-      name: 'insert-dependency-links',
+      name: 'add-prefetch',
```

Comment:
similar to the above comment,  I think "insert" is meaningful here. Maybe `insert-prefetch-links`?